### ISSUE TITLE
Auto update check and official Lulu website still pointing to 2.5.1 as being the latest version

### ISSUE THREAD

**Issue by fsafq41** [objective-see/LuLu#589]
Any reason for this, while the latest version on GitHub is said to be 2.6.2?

![Screen Shot 2024-02-16 at 12 37 13 PM](https://github.com/objective-see/LuLu/assets/129561979/d8cb6984-739b-4478-83a0-cc0e58650d60)

![Screen Shot 2024-02-16 at 12 37 36 PM](https://github.com/objective-see/LuLu/assets/129561979/62585fa6-2a92-4eef-9e05-edb72a633822)

**Comment by Chasapple4** [comment]
Current Lulu is  macOS 10.15+
Are on on 10.15 or higher?

**Comment by fsafq41** [comment]
> Current Lulu is macOS 10.15+ Are on on 10.15 or higher?

Definitely way higher than 10.15. Lulu is on 3 macs with mix of Apple Silicon/Intel on Sonoma, Ventura and Monterey.
Auto update doesn't work on any of them. In fact contrary to other objective-see applications, I don't remember Lulu's auto update having ever worked since starting using it many years ago.

Btw still the same issue with the latest version:
even though you released 2.6.3 a week ago, auto update is still saying that 2.6.2 is the latest version...

![Screen Shot 2024-03-14 at 9 22 15 PM](https://github.com/objective-see/LuLu/assets/129561979/d10d0b56-122c-4a4d-8e28-313755034dd7)

**Comment by fsafq41** [comment]
I highly recommend this app:
[https://github.com/mangerlahn/latest](url)
"Latest" checks if all your apps are up to date, including Lulu.
In fact it is how I find out about Lulu's newer versions as its auto-update is broken.

**Comment by Chasapple4** [comment]
> I highly recommend this app:
> [https://github.com/mangerlahn/latest](url)
> "Latest" checks if all your apps are up to date, including Lulu.
> In fact it is how I find out about Lulu's newer versions as its auto-update is broken.

Does this have anything to do with this open issue?

**Comment by fsafq41** [comment]
> > I highly recommend this app:
> > [https://github.com/mangerlahn/latest](url)
> > "Latest" checks if all your apps are up to date, including Lulu.
> > In fact it is how I find out about Lulu's newer versions as its auto-update is broken.
> 
> Does this have anything to do with this open issue?

Yes absolutely, this is an alternative way to address this long standing open issue

**Comment by Chasapple4** [comment]
> > > I highly recommend this app:
> > > [https://github.com/mangerlahn/latest](url)
> > > "Latest" checks if all your apps are up to date, including Lulu.
> > > In fact it is how I find out about Lulu's newer versions as its auto-update is broken.
> > 
> > 
> > Does this have anything to do with this open issue?
> 
> Yes absolutely, this is an alternative way to address this long standing open issue

Many macOS apps use Sparkle framework for update checks, that app is an overall update checker.

Sometimes auto updates are delay in a roll out phase.
